manage.py

```diff
@@ -29,42 +29,65 @@ def seed_categories():
 
     food = Category.get_by_name("Food")
     if not food:
-        food = Category("Food", "", 0)
+        food = Category()
+        food.name = "Food",
+        food.description = ""
+        food.sort_order = 0
         food.save()
-    shelter = Category.get_by_name("Shelter")
-    if not shelter:
-        shelter = Category("Shelter", "Housing for a limited time.", 0)
-        shelter.save()
-    clothing = Category.get_by_name("Clothing")
-    if not clothing:
-        clothing = Category("Clothing", "Shirts, shoes, and all things.", 0)
-        clothing.save()
-    other = Category.get_by_name("Other")
-    if not other:
-        other = Category("Other", "Other help that doesn't fit elsewhere", 0)
-        other.save()
-
-    return (food, shelter, clothing, other)
+    food_test1 = Service.get_by_name("Burritos")
+    if not food_test1:
+        food_burritos = Service(
+            name="Burritos", description="Filler", category=food
+        )
+        food_burritos.save()
+
+#    shelter = Category.get_by_name("Shelter")
+#    if not shelter:
+#        shelter = Category()
+#        shelter.name = "Shelter"
+#        shelter.description = "Housing for a limited time."
+#        shelter.sort_order = 0
+#        shelter.save()
+#    shelter_one_night = Service.get_by_name("One Night")
+#    if not shelter_one_night:
+#        shelter_one_night = Service(
+#            name="One Night", description="Need a pillow for the right?",
+#            category=shelter
+#        )
+#        shelter_one_night.save()
+
+#    clothing = Category.get_by_name("Clothing")
+#    if not clothing:
+#        clothing = Category()
+#        clothing.name = "Clothing"
+#        clothing.description = "Shirts, shoes, and all things."
+#        clothing.sort_order = 0
+#        clothing.save()
+#    clothing_shirts = Service.get_by_name("Shirts")
+#    if not clothing_shirts:
+#        clothing_shirts = Service(
+#            name="Shirts", description="Shirts of all types and sizes",
+#            category=clothing
+#        )
```

Comment:
Agreed, that was for a work in progress. Absolutely would not be wanting that in the main branch.